dataset_visualization.py

```diff
@@ -26,6 +27,7 @@ def play_files_parallel(td_files, labels=None, delta_t=50000, skip=0):
     box_videos = [PSEELoader(glob(td_file.split('_td.dat')[0] +  '*.npy')[0]) for td_file in td_files]
 
     height, width = videos[0].get_size()
+    labelmap = vis.LABELMAP if height == 240 else vis.LABELMAP_LARGE
```

Comment:
not obvious for me... could you explain why you use different labels for large frames than for smaller frame?

Reply:
we have to public detection datasets with different label maps and this is a handy way to distinguish between them.

for me this is not a general solutions but good enough for now on

Reply:
ok, got it!